homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -288,9 +282,10 @@ def parse_hue_api_put_light_body(request_json, entity):
         # Make sure the entity actually supports brightness
         entity_features = entity.attributes.get(ATTR_SUPPORTED_FEATURES, 0)
 
-        if (entity_features & SUPPORT_BRIGHTNESS) == SUPPORT_BRIGHTNESS:
-            report_brightness = True
-            result = (brightness > 0)
+        if entity.domain == "light":
```

Comment:
This is actually more than just cosmetic. This was causing the fan component to enter the wrong branch. Comparisons against supported features are only valid within their domain.